libraries/testing/tester.cpp

```diff
@@ -529,8 +529,13 @@ namespace eosio::testing {
    signed_block_ptr base_tester::produce_blocks( uint32_t n, bool empty ) {
       signed_block_ptr res;
       if( empty ) {
-         for( uint32_t i = 0; i < n; ++i )
+         for( uint32_t i = 0; i < n; ++i ) {
```

Comment:
Skip voting only when producing empty blocks?

Reply:
For some tests you need to vote on every block, for example the block log tests that require blocks to be finalized. This just adds additional analogous functionality to "empty".